pallets/collator-allowlist/src/benchmarking.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "runtime-benchmarks")]
+use super::*;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+
+benchmarks! {
+	// Add a collator to the allowlist
+	add {
+		let collator_id = get_account::<T>();
+	}: add(RawOrigin::Root, collator_id.clone())
+	verify {
+		assert!(<Allowlist<T>>::get(collator_id).is_some(), "Collator should be in the allowlist");
+	}
+
+	// Remove a collator from the allowlist
+	remove {
+		let collator_id = get_account::<T>();
+		// We need the collator to already be in the allowlist before we remove it.
+		<Allowlist<T>>::insert(collator_id.clone(), ());
+	}: remove(RawOrigin::Root, collator_id.clone())
+	verify {
+		assert!(<Allowlist<T>>::get(collator_id).is_none(), "Collator should have been removed");
+	}
+}
+
+impl_benchmark_test_suite!(Pallet, crate::mock::new_test_ext(), crate::mock::Test,);
+
+// Return an account that will be included as part of the initial
+// state of the pallet_session in the runtime used for benchmarking.
+fn get_account<T: Config>() -> T::ValidatorId {
+	let pub_key: [u8; 32] = [
```

Comment:
create a new account using benchmarking utils instead of static ones.

Reply:
I followed the example of other pallet's benchmarking and in this case we do need to get this specific account. Could you also expand on what do you refer to as "benchmarking utils"?

Reply:
https://github.com/centrifuge/centrifuge-chain/blob/tocc/pallets/loan/src/benchmarking.rs#L133
Here is how you can generate an account coming from benchmarking utils

Reply:
@vedhavyas that looks interesting but I am not able to generate the exact same pub key that we need to make this work. I have tried `frame_benchmarking::account::<T::AccountId>("/Alice", 0, 0);` and `frame_benchmarking::account::<T::AccountId>("//Alice", 0, 0);` but both fail to match with the key used in the runtime we benchmark against.

Reply:
hm I would have to check but I wasn't aware of the requirement for it to be predefined account.